js_modules/dagit/packages/core/src/ui/Tooltip.tsx

```diff
@@ -16,15 +16,24 @@ export const GlobalTooltipStyle = createGlobalStyle`
       color: ${ColorsWIP.Gray50};
       padding: 8px 16px;
   }
+
+  .block-tooltip.bp3-popover2-target {
+    display: block;
+  }
 `;
 
 // Overwrite arrays instead of concatting them.
 const overwriteMerge = (destination: any[], source: any[]) => source;
 
-export const Tooltip: React.FC<Tooltip2Props> = (props) => (
-  <Tooltip2
+interface Props extends Tooltip2Props {
+  display?: React.CSSProperties['display'];
+}
```

Comment:
The default `display: inline-block` on the blueprint Tooltip wrapper has been driving me crazy. Adding an escape hatch here.